src/components/donation/Steps/Splits/Splits.tsx

```diff
@@ -51,15 +51,15 @@ export default function Split({
       {/** percentages */}
       <div className="flex justify-between mt-6 mb-3">
         <div className="grid gap-1 grid-cols-[auto_1fr] items-center">
-          <Image src={leaf} className="mr-1" />
-          <p className="text-xl">{lockedSplitPct}%</p>
-          <p className="uppercase text-xs col-span-full">Sustainable fund</p>
-        </div>
-        <div className="grid gap-1 grid-cols-[1fr_auto] items-center">
           <p className="text-xl text-right">{liqSplitPct}%</p>
           <Image src={sendMoney} className="ml-1" />
           <p className="uppercase text-xs col-span-full">Instantly Available</p>
         </div>
+        <div className="grid gap-1 grid-cols-[1fr_auto] items-center">
```

Comment:
```suggestion
        <div className="grid gap-1 grid-cols-[auto_1fr] items-center">
```
and the other container (liquid) be `[1fr_auto]`

request change: to get the intended middle alignment
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/ce4c752c-9cb8-4176-980c-8a11653f0971)

comment: personally, alignment looks off to me though, prefers everythng aligned against the container
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/bd5e4b76-3b4c-4b08-9b38-b7d60f1df7fb)

Reply:
@ap-justin Good suggestion. I will run this idea by Tejaswini. Can always make this change separately though in new tiny ticket, as it's unrelated to the fix at hand (ie. swap the labels).

Reply:
@SovereignAndrey, requested changes to fix icon alignment 

> request change: to get the intended middle alignment

![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/88be01b5-509e-4859-a1ec-a79716b5755a)

```suggestion
        <div className="grid gap-1 grid-cols-[auto_1fr] items-center">
```
> and the other container (liquid) be `[1fr_auto]`

Reply:
Sorry, closed that too quickly w/o looking closely enough at the fix and only the images and second block of text about the container alignment. :+1:

Reply:
@ap-justin Fixed by reverting col grid change (see: [ec7a9d5](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2816/commits/ec7a9d516734f5f6699f35a91f30b6b7aaf0f99b))